programs/voter-stake-registry/src/instructions/clawback.rs

```diff
@@ -68,6 +68,16 @@ pub fn clawback(ctx: Context<Clawback>, deposit_entry_index: u8) -> Result<()> {
         // Load the accounts.
         let registrar = &ctx.accounts.registrar.load()?;
         let voter = &mut ctx.accounts.voter.load_mut()?;
+
+        // Governance may forbid clawback which is a form of withdraw, for example when engaged
+        // in a vote.
+        let token_owner_record = voter.load_token_owner_record(
```

Comment:
maybe this isnt correct, revert patch and add a note that clawback should go ahead despite user been engaged in an active proposal?

Reply:
ok added a note